ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -207,6 +207,19 @@ tests:
     test:
     - chain: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-shared-vpc-auto
+- as: aws-rosa-sts-shared-vpc-upgrade-e2e-f7
+  cron: 34 4 5,12,19,26 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      AVAILABLE_UPGRADE: "yes"
+      CHANNEL_GROUP: candidate
+      OPENSHIFT_VERSION: "4.13"
+      UPGRADED_TO_VERSION: "4.15"
```

Comment:
@xueli181114 @yasun1 What's the proper upgrade strategy  for shared-vpc cluster? in the CI, the [latest job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/49847/rehearse-49847-periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-shared-vpc-upgrade-e2e-f7/1772847789588353024) showed:

```
27-03-2024T07:41:58  Upgrade the cluster 2a8liv5v33gdf2hevlknq5e6u7ird8pr to the openshift version 4.14.18 successfully
27-03-2024T07:41:59  Error: No recommended 4.15 version for the cluster 2a8liv5v33gdf2hevlknq5e6u7ird8pr to be upgraded to.
```

Reply:
Let us just do upgrading the cluster from 4.14 to 4.15 for the single path upgrade. It is too hard to find a right path for the multi-path upgrade by the current logic.

Reply:
@yasun1, enabled `4.14.5` -> `4.15`.